### PR TITLE
[chore] RxWebSocket & WalletSDKRelayEventManager

### PR DESCRIPTION
**Pull request by erin-at-work**
### _Summary_

- Update node support versions
- Increase code coverage threshold
- Tests for:
  - WalletSDKRelayEventManager
  - RxWebSocket
image
image

### _How did you test your changes?_

- just tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,6 +108,6 @@
     "whatwg-fetch": "^3.5.0"
   },
   "engines": {
-    "node": ">= 10.0.0"
+    "node": ">= 14.0.0"
```
Comment (cb-jake) [inline package.json:111]: Would this constitute a major breaking change?

**Review by cb-jake — APPROVED** [review]
LGTM, I know I suggested the engines update but I'm curious if we should update it since it could be a breaking change for other apps.

File: package.json (hunk 1 above)
Comment (erin-at-work) [reply package.json:111]: It's likely, 🤔  
cc @ljharb

**Review by cb-jake — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ljharb) [reply package.json]: Yes, absolutely and always. why is this needed?

**Pull request merged**